h2o-py/h2o/model/model_base.py

```diff
@@ -454,6 +454,44 @@ def feature_interaction(self, max_interaction_depth=100, max_tree_depth=100, max
             
             return json['feature_interaction']
         print("No calculation available for this model")
+        
+        
+    def h(self, frame, variables):
+        """
+        Calculates Friedman and Popescu's H statistics, in order to look for interactions among variables in h2o gradient-boosting models
+        
+        NaN is returned if a computation is spoiled by weak main effects and rounding errors.
+        H varies from 0 to 1. The larger H, the stronger the evidence for an interaction among the variables.
+        
+        See Jerome H. Friedman and Bogdan E. Popescu, 2008, "Predictive learning via rule ensembles", *Ann. Appl. Stat.*
+        **2**:916-954, http://projecteuclid.org/download/pdfview_1/euclid.aoas/1223908046, s. 8.1.
+
+        
+        :param frame: the frame that current model has been fitted to
+        :param variables: variables of the interest
+        :return: H statistic of the variables 
+        
+        :examples:
+        >>> prostate_train = h2o.import_file("https://s3.amazonaws.com/h2o-public-test-data/smalldata/logreg/prostate_train.csv")
+        >>> prostate_train["CAPSULE"] = prostate_train["CAPSULE"].asfactor()
+        >>> gbm_h2o = H2OGradientBoostingEstimator(ntrees=100, learn_rate=0.1,
+        >>>                                 max_depth=5,
+        >>>                                 min_rows=10,
+        >>>                                 distribution="bernoulli")
+        >>> gbm_h2o.train(x=list(range(1,prostate_train.ncol)),y="CAPSULE", training_frame=prostate_train)
+        >>> h = gbm_h2o.h(prostate_train, ['DPROS','DCAPS'])
+        """
+        supported_algos = ['gbm', 'xgboost']
```

Comment:
fiy @sebhrusen is working on a solution for this problem - you might want to revise the implementation after he is done (in master)